index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Laika Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+  <section id = "main">
+     <h1 id ="title"> Laika</h1>
+     <p class="subtitle">The first animal in space</p>
+     <figure id = "img-div">
+       <img id ="image"  src="https://cdn.mos.cms.futurecdn.net/VwRStV56Xdxstz5UrZDsYA-1200-80.jpg">
+       <p id = img-caption>laika's Picture inside the sputnik capsule</p>
+     </figure>
+     <p id = "tribute-info"> <span> Laika </span> was a Soviet space dog who became one of the first animals in space, and the first animal to orbit the Earth. Laika, a stray mongrel from the streets of Moscow, was selected to be the occupant of the Soviet spacecraft Sputnik 2 that was launched into low orbit on 3 November 1957.
+     </p>
+     <h2>Here's a time line of Laika's life</h2>
+     <ul>
+       <li><span>1954</span> - born in Moscow, Soviet Union</li>
+       <li> <span>1957</span> - Laika became the first animal launched into Earth orbit, paving the way for human spaceflight during the upcoming years. </li>
+       <li> <span>1957</span> - laika  died of overheating or asphyxiation shortly before she was to be remotely euthanized via poison.</li>
+       <li> <span>2002</span> - The true cause and time of her death were made public  </li>
+      <li> <span>2008</span> - Russian officials unveiled a monument to Laika. A small monument in her honour was built near the military research facility in Moscow that prepared Laika's flight to space. </li>
+     </ul>
+
+     <p class="quote">
+       "The more time passes, the more I'm sorry about it. We shouldn't have done it. We did not learn enough from the mission to "
+     </p>
+     <span class="credit">Turkina  one of the scientists assigned to Laika's program</span>
+    
+     <a id = "tribute-link" target = "_blank" href="https://en.wikipedia.org/wiki/Laika"> Read more </a>
+  </section>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
I think it would be better to remove those script lines 
there is no use for them at this stage...
Clean-Code should be a habit.